src/CSM.js

```diff
@@ -21,7 +21,8 @@ export default class CSM {
 		this.lightIntensity = data.lightIntensity || 1;
 		this.lightNear = data.lightNear || 1;
 		this.lightFar = data.lightFar || 2000;
-		this.lightMargin = data.lightMargin || 200;
+        this.lightMargin = data.lightMargin || 200;
+        this.customSplitsCallback = data.customSplitsCallback || undefined;
```

Comment:
`|| undefined` doesn't make sense

Reply:
Oh yeah, if the user doesn't pass anything it's automatically undefined right?
So I'll just remove it.

Reply:
Yeah, just remove it.